### DISCUSSION TITLE
Using $this when not in object context when using Octane

### DISCUSSION THREAD

**Discussion by vmiguellima** [livewire/livewire#5293] [7. Bugs (archived)] [answered]
Livewire when in production throws `Using $this when not in object context`.

My application is running on vapor. It was working as normal, and all the sudden it throws: `Using $this when not in object context`.

Has far I can see, this is related to pagination.

On localhost works fine.

Error thrown:
```php

"exception": {
  "class": "Spatie\LaravelIgnition\Exceptions\ViewException",
  "message": "Using $this when not in object context",
  "code": 0,
  "file": "/var/task/resources/views/vendor/livewire/tailwind.blade.php:3",
  "previous": {
  "class": "Error",
  "message": "Using $this when not in object context",
  "code": 0,
  "file": "/tmp/storage/framework/views/2d67d537564cd810c86df041186d40d211c32fdf.php:3"
  }
}
```

**Comment by joshhanley** [comment, marked as answer]
@vmiguellima yep there is currently an issue with Livewire pagination and Octane, see #5130 for more details.

**Comment by joshhanley** [comment]
There is currently a work around for this here until we can get it fixed https://github.com/livewire/livewire/discussions/5351#discussioncomment-4091512

Hope this helps!
